Frameworks/UIKit/UITapGestureRecognizer.mm

```diff
@@ -291,7 +281,10 @@ - (void)touchesEnded:(NSSet*)touches withEvent:(UIEvent*)event {
                 _wasRecognized = true;
             } else {
                 _state = UIGestureRecognizerStateRecognized;
-                [[self class] _failActiveExcept:self];
+
+                // TODO: we should revisit. Fail all other active UITapGestureRecognizer should not
```

Comment:
#1961

---
In reply to: [100935659](https://github.com/Microsoft/WinObjC/pull/1909#discussion_r100935659) [](ancestors = 100935659)